### ISSUE TITLE
Adding stdlibs breaks test-revise-*

### ISSUE THREAD

**Issue by Keno** [JuliaLang/julia#38564]
```
$ make test-revise-FileWatching
    JULIA test/revise-FileWatching
ERROR: LoadError: no Revise.track recipe for module LazyArtifacts
Stacktrace:
```

I'm sure this'll be fixed by upgrading Revise, but it seems like this should just issue a warning instead. cc @timholy

**Comment by timholy** [comment]
Any objections to #38450?

**Comment by Keno** [comment]
Fixed by #38450
